### ISSUE TITLE
Support the rewriting of react widgets for Cerner needs

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#7860]
## User Story or Problem Statement

As a Veteran, i want my top tasks to direct me towards the appropraite health records management system (Cerner, MyHealtheVet), so that i can accomplish my task. 

As a react widget developer, i need a source of truth for what facilities are Cerner, Cerner staged, or VistA, so that the react widget can present a different experience for Veterans. 

Stretch (spin this out to a separate issue if it's not finished:  As a react widget developer, i need a source of truth for what health care systems are Cerner, Cerner staged, or VistA, so that the react widget can present a different experience for Veterans.

Here is the [rough idea](https://docs.google.com/drawings/d/1bx-kyoAmNjtReu8turBIOokyEpW7TCi9fTjxMr6-mfw/edit?usp=sharing).
![image](https://user-images.githubusercontent.com/5752113/152368710-9934f0ff-3f5d-458e-9092-e7ceebbf6cb2.png)

## Acceptance Criteria
- [ ] TBD

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [x] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected.
- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [x] `⭐️ Public Websites`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

**Comment by cmaeng** [comment]
blocked by an internal meeting between Sitewide CMS and Platform CMS teams. looking for that meeting to happen the week of 3/28/22.

**Comment by cmaeng** [comment]
Other MyVA portal work is looking for Cerner data per Dave Conlon. tentative timeline: April, for Columbus go-live to Cerner 4/30, which could be done "the Spokane and Walla Walla way". Roseburg is next, 6/11

**Comment by mchelen-gov** [comment]
Regarding CMS Platform team scope:
- Responsible for shared services used across all VFS teams such as GraphQL API
- Responsible for defining guidance on external integration approach and documentation (https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/READMES/upstream-dependencies.md, https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/READMES/downstream_dependencies.md)

Please advise if guidance is needed from Platform CMS on these topics listed. From the description here the current features of GraphQL API with Content Build are sufficient for VFS teams to implement this product specific use case.

**Comment by cmaeng** [comment]
@mchelen-gov not sure where you're pulling the Platform CMS scope from but based on this [GH page](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms) and the latest [RASCI](https://docs.google.com/spreadsheets/d/1ezikQhrTCsX36j91ACyf4np_StQdRsoQUmRMN3EmDk8/edit#gid=11057317) this feels very much within Platform CMS's purview given that it's:

**Comment by cmaeng** [comment]
@mchelen-gov not sure where you're pulling the Platform CMS scope from but based on this [GH page](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms) and the latest [RASCI](https://docs.google.com/spreadsheets/d/1ezikQhrTCsX36j91ACyf4np_StQdRsoQUmRMN3EmDk8/edit#gid=11057317) this feels very much within Platform CMS's purview given that it's:

- touching how content is consumed architecturally
- a general content availability need that isn't local to a single product

is your understanding of the scope only to entail things that are shared across all VFS teams and this is too limited in that way?

**Comment by cmaeng** [comment]
@mchelen-gov, also as mentioned in Slack, Sitewide CMS team is undergoing a significant amount of organizational change next sprint with several high priority initiatives in flight. This request is also an effort to get one of those high priority things (Cerner) across the finish line collectively as a CMS team. cc: @davidconlon

**Comment by wesrowe** [comment]
Based on learnings from #8888, closing this issue in favor of two others that we learned had been written and vetted by tech and product leadership: #8199 and #8201. This approach will make the task of providing CMS data to the front-end via static json files much easier in the future.
